packages/pangea-sdk/.sdk-ci.yml

```
@@ -48,9 +49,47 @@ variables:
   parallel:
     matrix:
       - CLOUD: [AWS, GCP]
-
+        ENV: ${SERVICE_AUDIT_ENV}
+        TEST: ITAuditTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_AUTHN_ENV}
+        TEST: ITAuthNTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_EMBARGO_ENV}
+        TEST: ITEmbargoTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_DOMAIN_INTEL_ENV}
+        TEST: ITDomainIntelTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_IP_INTEL_ENV}
+        TEST: ITIPIntelTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_URL_INTEL_ENV}
+        TEST: ITURLIntelTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_USER_INTEL_ENV}
+        TEST: ITUserIntelTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_FILE_INTEL_ENV}
+        TEST: ITFileIntelTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_FILE_SCAN_ENV}
+        TEST: ITFileScanTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_REDACT_ENV}
+        TEST: ITRedactTest
+      - CLOUD: [AWS, GCP]
+        ENV: ${SERVICE_VAULT_ENV}
+        TEST: ITVaultTest
   rules:
-    - if: $CI_PIPELINE_SOURCE == "push"
+    - if: '$CLOUD == "GCP" && $TEST == "ITFileScanTest"'
+      allow_failure: true
+    - if: '$CLOUD == "GCP" && $TEST != "ITFileScanTest"'
+      allow_failure: false
+    - if: '$CLOUD != "GCP"'
+      allow_failure: false
```

Now I wonder if this would work:

```suggestion
    - if: '$CLOUD == "GCP" && $TEST == "ITFileScanTest"'
      allow_failure: true
    - if: 'true'
      allow_failure: false
```
but since I'm not sure, and what you have works, I'll approve anyways.

---

I guess it should work, weird that it doesn't use default value...